src/Microsoft.ML/CSharpApi.cs

```diff
@@ -406,6 +490,30 @@ public void Add(Microsoft.ML.Trainers.AveragedPerceptronBinaryClassifier input,
                 _jsonNodes.Add(Serialize("Trainers.AveragedPerceptronBinaryClassifier", input, output));
             }
 
+            public Microsoft.ML.Trainers.BinaryClassifierEnsemble.Output Add(Microsoft.ML.Trainers.BinaryClassifierEnsemble input)
```

Comment:
>BinaryClassifierEnsemble [](start = 41, length = 24)

The trainers seem to follow the convention of having a *suffix* of `BinaryClassifier`. Do we want to adhere to that? If not, why not? This has it as a prefix. Is that intentional?